.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -1,18 +1,32 @@
+---
 name: Report an issue
-description: Tell us about something that's not working the way we (probably) intend
+description:
+  Tell us about something that is not working the way we (probably) intend.
 body:
-  - type: input
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to Elixir.
```

Comment:
```suggestion
        Thank you for contributing to Elixir! :heart:
```

Reply:
We need all the hearts! 😄

Reply:
how about a purple heart instead? :purple_heart: